File: 01-business-sub-system/src/main/java/com/github/yingzhuo/datawarehouse/businesssubsys/dao/OrderDao.scala
package com.github.yingzhuo.datawarehouse.businesssubsys.dao

import com.github.yingzhuo.datawarehouse.businesssubsys.domain.Order
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
trait OrderDao extends JpaRepository[Order, String] {
}

File: 01-business-sub-system/src/main/java/com/github/yingzhuo/datawarehouse/businesssubsys/dao/OrderDao.java
package com.github.yingzhuo.datawarehouse.businesssubsys.dao;

import com.github.yingzhuo.datawarehouse.businesssubsys.domain.Order;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface OrderDao extends JpaRepository<Order, String> {
}
